js_modules/dagster-ui/packages/ui-core/src/graph/SVGViewport.tsx

```diff
@@ -224,26 +225,31 @@ const PanAndZoomInteractor: SVGViewportInteractor = {
               }}
             />
           </Box>
-          <IconButton
-            onClick={() => {
-              const x = viewport.element.current!.clientWidth / 2;
-              const y = viewport.element.current!.clientHeight / 2;
-              const scale = Math.max(
-                viewport.getMinZoom(),
-                viewport.state.scale - BUTTON_INCREMENT,
-              );
-              viewport.adjustZoomRelativeToScreenPoint(scale, {x, y});
-            }}
-          >
-            <Icon size={24} name="zoom_out" />
-          </IconButton>
+          <Tooltip content="Zoom out">
+            <IconButton
+              style={{borderTopLeftRadius: 0, borderTopRightRadius: 0}}
+              onClick={() => {
+                const x = viewport.element.current!.clientWidth / 2;
+                const y = viewport.element.current!.clientHeight / 2;
+                const scale = Math.max(
+                  viewport.getMinZoom(),
+                  viewport.state.scale - BUTTON_INCREMENT,
+                );
+                viewport.adjustZoomRelativeToScreenPoint(scale, {x, y});
+              }}
+            >
+              <Icon size={24} name="zoom_out" />
+            </IconButton>
+          </Tooltip>
         </Box>
         <Box flex={{direction: 'column', alignItems: 'center', gap: 8}} margin={{top: 8}}>
           {viewport.props.additionalToolbarElements}
           <Box>
```

Comment:
This can just be a `div`. (Or you might not even need it, depending on how it renders with the `Tooltip` wrapper as the flex child.)